src/components/http/Request.js

```diff
@@ -740,6 +740,13 @@ class Request {
         return this._body;
     }
 
+    /**
+     * Sets the req.body value
+     */
+    set body(value) {
+        this._body = value
```

Comment:
For this, add an if statement check before setting the _body property which essentially asserts that If we have some _body value (meaning we have some type of body content) then the parameter value cannot be an empty/null value. You can likely just do truthy comparisons such as if (this._body && !value) throw.

Reply:
I added it in two conditions so you can get a more specific error message.